install/install.md

```diff
@@ -47,6 +47,16 @@ Use the selector tool below to select your preferred method, packages, and envir
 ## Installation Troubleshooting
 
 ### **Conda Issues**
+<i class="fas fa-info-circle"></i> CUDA 12.0 arm packages don't exist:<br/>
+Conda-forge doesn't yet support the minimum required glibc (2.27) for CUDA 12 on arm. For arm support, please use an 11.x installation.
+
+<i class="fas fa-info-circle"></i> CUDA 12.0 packages don't work with stable PyTorch packages: <br/>
+PyTorch currently only has nightly builds for CUDA 12.1, stable builds are limited to CUDA 11.x. <br/>
+The installation method below may function if there is a hard-requirement of CUDA 12 but it is currently unsupported:
+```
+mamba create -n rapids-pytorch-cu12 -c rapidsai -c pytorch-nightly -c conda-forge -c nvidia cuml pytorch pytorch-cuda=12.1 cuda-version=12.0
```

Comment:
We want to install all of RAPIDS and not just cuML, right? Also do we want a version pinning to try and ensure that things are fetched properly?
```suggestion
mamba create -n rapids-pytorch-cu12 -c rapidsai -c pytorch-nightly -c conda-forge -c nvidia rapids=23.08 cuda-version=12.0 pytorch pytorch-cuda=12.1
```

Reply:
Is the install page able to access the site data like the selector? ie could I do:

`mamba ... rapids={{ site.data.releases.stable.version }} ...`

Reply:
I don’t know. Please try it! Let’s find out.

Reply:
We sure can!

Reply:
Awesome. I wonder how many places we've missed the opportunity to do this in the past... (though we have some places where it makes more sense to hardcode if the guidance is temporary and applicable only to one specific release).